test/pom.xml

```diff
@@ -30,6 +30,7 @@
   <name>Apache Accumulo Testing</name>
   <description>Tests for Apache Accumulo.</description>
   <properties>
+    <accumulo.module.name>org.apache.accumulo</accumulo.module.name>
```

Comment:
This one doesn't follow the convention. Should this be `org.apache.accumulo.test`?

Reply:
Yeah I think I missed this one.  Good catch, thanks.